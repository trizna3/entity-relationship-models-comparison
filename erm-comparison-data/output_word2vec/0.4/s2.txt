budovy-budova
platby-poplatok
izby-izba
studenti-EMPTY
konta-konto
zmluvy-zmluva
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-konto-ubytovani
