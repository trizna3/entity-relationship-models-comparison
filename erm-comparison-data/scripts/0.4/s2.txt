entity set internat {};

entity set budova {
	"poschodie"
	"adresa"
};

entity set izba {
	"kapacita"
};

entity set zmluva {
	"zaciatok"
	"koniec"
};

entity set poplatok {
	"suma"
	"cas"
};

entity set ubytovani {
	"konto"
};

association between 
{
	internat ONE
	budova MANY
}{};

association between 
{
	budova ONE 
	izba MANY
}{};

association between 
{
	izba ONE
	poplatok ONE
}{};

association between 
{
	izba ONE 
	ubytovani ONE
}{};

association between 
{
	ubytovani ONE
	zmluva ONE
}{};