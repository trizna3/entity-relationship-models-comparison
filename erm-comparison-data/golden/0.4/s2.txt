budovy-budova
izby-izba
zmluvy-zmluva
platby-poplatok
studenti-ubytovani
konta-konto
CONTRACT_11_ASSOCIATION:E-konta-studenti
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-konto-ubytovani